### DISCUSSION TITLE
Brightness, don´t get it :(

### DISCUSSION THREAD

**Discussion by UweNB** [Foddy/node-red-contrib-huemagic#257] [Q&A]
i'm a total beginner and don't know much about node red. sorry about that
currently i try to change the brightness of a light. i pass the msg.payload.brightness =100 to the "hue light"-node - but the light stays at the current value.
can someone show me a little flow where i can see what i am doing wrong? thanks!

**Comment by Christoph-87** [comment]
The flow should look like int the readme in the section "Hue Lights". It is important that the inject node either auto injects after deploying the changes or that you click it afterwards to trigger it. I assume you have set the msg.payload.brightness in there, too. When you have the debug node behind, you can see if it was triggered at all.

**Reply by UweNB** [reply]
could someone kindly upload a short flow, where you can see the function even as a beginner? the purely textual description in the readme does not really help me. thanks!

**Reply by Christoph-87** [reply]
I just saw @Foddy added example Flows to the readme. The example flows can be added directly from Node Red or you use the json under /examples/Hue%20Light.json. This also captures brightness.

**Reply by Foddy** [reply]
Exactly :)

The examples are now full featured and you should be able to get the information that you need for your issue in the latest version :)

**Comment by UweNB** [comment]
Thx for hint! 
Especially for beginners flows are better to understand than "look in the documents" :)
@Foddy: thanks for all the work incl. documentation that you have done here!

**Comment by UweNB** [comment]
Hallo Foddy, mach mal in deutsch weiter, wenn es okay ist ...

Hatte mir den unten angehängten Flow gebastelt, teils mit Hilfe aus dem Forum. Nachdem ich heute Deiner 4er Update installiert hatte, geht  es nicht mehr, obwohl ja bei den Hue-Steckdosen weiterhin true=ein und false gleich aus ist. Hast Du eine Idee? Wenn ich die Injekt-Nodes betätige, funktioniert es, per Fernbedienung nicht ... ist ja nix anderes als vorher - oder? Hier der Flow:

`[
    {
        "id": "2c8dde1dc9576b81",
        "type": "function",
        "z": "3dc0134f3805614a",
        "name": "",
        "func": "let on_off = msg.payload.substr(1);\nmsg.channel = msg.payload.substr(0,1);\nif(on_off ===\"on\"){\n    msg.payload = true;\n}else if(on_off === \"off\"){\n    msg.payload = false\n}else{\n    msg = null;\n}\nreturn msg;",
        "outputs": 1,
        "noerr": 0,
        "initialize": "",
        "finalize": "",
        "libs": [],
        "x": 680,
        "y": 660,
        "wires": [
            [
                "db8abab30126e2f5"
            ]
        ]
    },
    {
        "id": "db8abab30126e2f5",
        "type": "switch",
        "z": "3dc0134f3805614a",
        "name": "",
        "property": "channel",
        "propertyType": "msg",
        "rules": [
            {
                "t": "eq",
                "v": "a",
                "vt": "str"
            },
            {
                "t": "eq",
                "v": "b",
                "vt": "str"
            },
            {
                "t": "eq",
                "v": "c",
                "vt": "str"
            },
            {
                "t": "eq",
                "v": "d",
                "vt": "str"
            }
        ],
        "checkall": "true",
        "repair": false,
        "outputs": 4,
        "x": 830,
        "y": 660,
        "wires": [
            [
                "4d8ed5379bd46483"
            ],
            [
                "b345380fad950493"
            ],
            [
                "94f80cd8f6632c03"
            ],
            [
                "dba2ec8cd905823f"
            ]
        ]
    },
    {
        "id": "eacd33d4c551d099",
        "type": "inject",
        "z": "3dc0134f3805614a",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "aoff",
        "payloadType": "str",
        "x": 330,
        "y": 460,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "d086b65059ec746e",
        "type": "inject",
        "z": "3dc0134f3805614a",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "aon",
        "payloadType": "str",
        "x": 330,
        "y": 420,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "4d8ed5379bd46483",
        "type": "hue-light",
        "z": "3dc0134f3805614a",
        "name": "TV Stedo",
        "bridge": "36b93868f3cfa2da",
        "lightid": "d5a5a8d2-f721-4478-ba95-d4f2d55d78a4",
        "colornamer": true,
        "skipevents": false,
        "initevents": false,
        "x": 1100,
        "y": 560,
        "wires": [
            []
        ]
    },
    {
        "id": "b345380fad950493",
        "type": "hue-light",
        "z": "3dc0134f3805614a",
        "name": "Wandlicht",
        "bridge": "36b93868f3cfa2da",
        "lightid": "7c618702-10e6-41fc-a8a8-ed6b33c518e0",
        "colornamer": true,
        "skipevents": false,
        "initevents": false,
        "x": 1100,
        "y": 620,
        "wires": [
            []
        ]
    },
    {
        "id": "8411484993ac4c0f",
        "type": "inject",
        "z": "3dc0134f3805614a",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "boff",
        "payloadType": "str",
        "x": 270,
        "y": 560,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "10afeb8cef5bec40",
        "type": "inject",
        "z": "3dc0134f3805614a",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "bon",
        "payloadType": "str",
        "x": 270,
        "y": 520,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "94f80cd8f6632c03",
        "type": "hue-light",
        "z": "3dc0134f3805614a",
        "name": "Weihnacht",
        "bridge": "36b93868f3cfa2da",
        "lightid": "fdd60a41-ede1-49bd-9cee-06bc3b244dee",
        "colornamer": true,
        "skipevents": false,
        "initevents": false,
        "x": 1110,
        "y": 680,
        "wires": [
            []
        ]
    },
    {
        "id": "dba2ec8cd905823f",
        "type": "hue-light",
        "z": "3dc0134f3805614a",
        "name": "Frei",
        "bridge": "36b93868f3cfa2da",
        "lightid": "2f4a6e5d-e049-4b29-84c0-682400a74810",
        "colornamer": true,
        "skipevents": false,
        "initevents": false,
        "x": 1090,
        "y": 740,
        "wires": [
            []
        ]
    },
    {
        "id": "810a14ba14b6c53b",
        "type": "inject",
        "z": "3dc0134f3805614a",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "coff",
        "payloadType": "str",
        "x": 350,
        "y": 660,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "4652db40c1ebcbc8",
        "type": "inject",
        "z": "3dc0134f3805614a",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "con",
        "payloadType": "str",
        "x": 350,
        "y": 620,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "7f639bd610427b4e",
        "type": "rfx-PT2262-in",
        "z": "3dc0134f3805614a",
        "name": "A-D",
        "port": "11cc440c.fcc32c",
        "deviceList": "0dd3cb0011ea3ee6",
        "topicSource": "all",
        "topic": "",
        "x": 350,
        "y": 340,
        "wires": [
            [
                "42af240be3328f2d"
            ]
        ]
    },
    {
        "id": "42af240be3328f2d",
        "type": "function",
        "z": "3dc0134f3805614a",
        "name": "",
        "func": "var interval = (370); // minimum interval between messages (ms)\ncontext.lastTime = context.lastTime || 0;\n\nvar now = Date.now();\n\nif (now-context.lastTime > interval) {\n  context.lastTime = now;\n  return msg;\n} else {\n  return null;\n}",
        "outputs": 1,
        "noerr": 0,
        "initialize": "",
        "finalize": "",
        "libs": [],
        "x": 540,
        "y": 340,
        "wires": [
            [
                "2c8dde1dc9576b81",
                "a126c8f2b8395e24"
            ]
        ]
    },
    {
        "id": "22a2650ee6d93384",
        "type": "inject",
        "z": "3dc0134f3805614a",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "doff",
        "payloadType": "str",
        "x": 230,
        "y": 780,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "14bcdacdbafa055c",
        "type": "inject",
        "z": "3dc0134f3805614a",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "don",
        "payloadType": "str",
        "x": 250,
        "y": 740,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "8a3720bdd399a9c6",
        "type": "inject",
        "z": "3dc0134f3805614a",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "30 16 * * *",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "con",
        "payloadType": "str",
        "x": 230,
        "y": 960,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "4c6359f5e1bf8a59",
        "type": "inject",
        "z": "3dc0134f3805614a",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "00 23 * * *",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "coff",
        "payloadType": "str",
        "x": 230,
        "y": 1000,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "a126c8f2b8395e24",
        "type": "debug",
        "z": "3dc0134f3805614a",
        "name": "",
        "active": true,
        "tosidebar": true,
        "console": false,
        "tostatus": false,
        "complete": "false",
        "statusVal": "",
        "statusType": "auto",
        "x": 810,
        "y": 520,
        "wires": []
    },
    {
        "id": "36b93868f3cfa2da",
        "type": "hue-bridge",
        "name": "HueMagic Bridge",
        "bridge": "192.168.1.23",
        "key": "IwnC3vTIm1b5YA9x1YgVMeFJ9MqvWBHPlJHFejnf",
        "disableupdates": false
    },
    {
        "id": "11cc440c.fcc32c",
        "type": "rfxtrx-port",
        "port": "/dev/ttyUSB0",
        "enableDebug": false
    },
    {
        "id": "0dd3cb0011ea3ee6",
        "type": "PT2262-device-list",
        "name": "Kanal A-D",
        "devices": [
            {
                "device": [
                    "PT2262"
                ],
                "payload": "aoff",
                "rawData": "0x104554",
                "pulseWidth": 350
            },
            {
                "device": [
                    "PT2262"
                ],
                "payload": "aon",
                "rawData": "0x104551",
                "pulseWidth": 350
            },
            {
                "device": [
                    "PT2262"
                ],
                "payload": "boff",
                "rawData": "0x105154",
                "pulseWidth": 350
            },
            {
                "device": [
                    "PT2262"
                ],
                "payload": "bon",
                "rawData": "0x105151",
                "pulseWidth": 350
            },
            {
                "device": [
                    "PT2262"
                ],
                "payload": "coff",
                "rawData": "0x105454",
                "pulseWidth": 350
            },
            {
                "device": [
                    "PT2262"
                ],
                "payload": "con",
                "rawData": "0x105451",
                "pulseWidth": 350
            },
            {
                "device": [
                    "PT2262"
                ],
                "payload": "doff",
                "rawData": "0x105514",
                "pulseWidth": 350
            },
            {
                "device": [
                    "PT2262"
                ],
                "payload": "don",
                "rawData": "0x105511",
                "pulseWidth": 350
            },
            {
                "device": [
                    "PT2262"
                ],
                "payload": "eon",
                "rawData": "0xda92e1",
                "pulseWidth": 350
            },
            {
                "device": [
                    "PT2262"
                ],
                "payload": "foff",
                "rawData": "0x415530",
                "pulseWidth": 350
            }
        ]
    }
]`

**Reply by Foddy** [reply]
Danke fürs Teilen des gesamten Flows. Ich schau mir das gleich mal an und finde hoffentlich den Fehler. Hört sich aber sehr mysteriös an, auch weil das ein relativ simpler Befehl eigentlich ist 🤔

**Reply by Foddy** [reply]
Könntest du bitte folgenden Flow mal importieren mir anschließend den Log aus der Sidebar hier posten?

```
[
    {
        "id": "900f2610476d2c9f",
        "type": "tab",
        "label": "Flow 1",
        "disabled": false,
        "info": "",
        "env": []
    },
    {
        "id": "2c8dde1dc9576b81",
        "type": "function",
        "z": "900f2610476d2c9f",
        "name": "",
        "func": "let on_off = msg.payload.substr(1);\nmsg.channel = msg.payload.substr(0,1);\nif(on_off ===\"on\"){\n msg.payload = true;\n}else if(on_off === \"off\"){\n msg.payload = false\n}else{\n msg = null;\n}\nreturn msg;",
        "outputs": 1,
        "noerr": 0,
        "initialize": "",
        "finalize": "",
        "libs": [],
        "x": 500,
        "y": 480,
        "wires": [
            [
                "db8abab30126e2f5"
            ]
        ]
    },
    {
        "id": "db8abab30126e2f5",
        "type": "switch",
        "z": "900f2610476d2c9f",
        "name": "",
        "property": "channel",
        "propertyType": "msg",
        "rules": [
            {
                "t": "eq",
                "v": "a",
                "vt": "str"
            },
            {
                "t": "eq",
                "v": "b",
                "vt": "str"
            },
            {
                "t": "eq",
                "v": "c",
                "vt": "str"
            },
            {
                "t": "eq",
                "v": "d",
                "vt": "str"
            }
        ],
        "checkall": "true",
        "repair": false,
        "outputs": 4,
        "x": 650,
        "y": 480,
        "wires": [
            [
                "4d8ed5379bd46483",
                "ee1dcffd99721703"
            ],
            [
                "b345380fad950493",
                "ee1dcffd99721703"
            ],
            [
                "94f80cd8f6632c03",
                "ee1dcffd99721703"
            ],
            [
                "dba2ec8cd905823f",
                "ee1dcffd99721703"
            ]
        ]
    },
    {
        "id": "eacd33d4c551d099",
        "type": "inject",
        "z": "900f2610476d2c9f",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "aoff",
        "payloadType": "str",
        "x": 150,
        "y": 280,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "d086b65059ec746e",
        "type": "inject",
        "z": "900f2610476d2c9f",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "aon",
        "payloadType": "str",
        "x": 150,
        "y": 240,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "4d8ed5379bd46483",
        "type": "hue-light",
        "z": "900f2610476d2c9f",
        "name": "TV Stedo",
        "bridge": "36b93868f3cfa2da",
        "lightid": "d5a5a8d2-f721-4478-ba95-d4f2d55d78a4",
        "colornamer": true,
        "skipevents": false,
        "initevents": false,
        "x": 920,
        "y": 380,
        "wires": [
            []
        ]
    },
    {
        "id": "b345380fad950493",
        "type": "hue-light",
        "z": "900f2610476d2c9f",
        "name": "Wandlicht",
        "bridge": "36b93868f3cfa2da",
        "lightid": "7c618702-10e6-41fc-a8a8-ed6b33c518e0",
        "colornamer": true,
        "skipevents": false,
        "initevents": false,
        "x": 920,
        "y": 440,
        "wires": [
            []
        ]
    },
    {
        "id": "8411484993ac4c0f",
        "type": "inject",
        "z": "900f2610476d2c9f",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "boff",
        "payloadType": "str",
        "x": 90,
        "y": 380,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "10afeb8cef5bec40",
        "type": "inject",
        "z": "900f2610476d2c9f",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "bon",
        "payloadType": "str",
        "x": 90,
        "y": 340,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "94f80cd8f6632c03",
        "type": "hue-light",
        "z": "900f2610476d2c9f",
        "name": "Weihnacht",
        "bridge": "36b93868f3cfa2da",
        "lightid": "fdd60a41-ede1-49bd-9cee-06bc3b244dee",
        "colornamer": true,
        "skipevents": false,
        "initevents": false,
        "x": 930,
        "y": 500,
        "wires": [
            []
        ]
    },
    {
        "id": "dba2ec8cd905823f",
        "type": "hue-light",
        "z": "900f2610476d2c9f",
        "name": "Frei",
        "bridge": "36b93868f3cfa2da",
        "lightid": "2f4a6e5d-e049-4b29-84c0-682400a74810",
        "colornamer": true,
        "skipevents": false,
        "initevents": false,
        "x": 910,
        "y": 560,
        "wires": [
            []
        ]
    },
    {
        "id": "810a14ba14b6c53b",
        "type": "inject",
        "z": "900f2610476d2c9f",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "coff",
        "payloadType": "str",
        "x": 170,
        "y": 480,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "4652db40c1ebcbc8",
        "type": "inject",
        "z": "900f2610476d2c9f",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "con",
        "payloadType": "str",
        "x": 170,
        "y": 440,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "7f639bd610427b4e",
        "type": "rfx-PT2262-in",
        "z": "900f2610476d2c9f",
        "name": "A-D",
        "port": "11cc440c.fcc32c",
        "deviceList": "0dd3cb0011ea3ee6",
        "topicSource": "all",
        "topic": "",
        "x": 170,
        "y": 160,
        "wires": [
            [
                "42af240be3328f2d"
            ]
        ]
    },
    {
        "id": "42af240be3328f2d",
        "type": "function",
        "z": "900f2610476d2c9f",
        "name": "",
        "func": "var interval = (370); // minimum interval between messages (ms)\ncontext.lastTime = context.lastTime || 0;\n\nvar now = Date.now();\n\nif (now-context.lastTime > interval) {\n context.lastTime = now;\n return msg;\n} else {\n return null;\n}",
        "outputs": 1,
        "noerr": 0,
        "initialize": "",
        "finalize": "",
        "libs": [],
        "x": 360,
        "y": 160,
        "wires": [
            [
                "2c8dde1dc9576b81",
                "a126c8f2b8395e24"
            ]
        ]
    },
    {
        "id": "22a2650ee6d93384",
        "type": "inject",
        "z": "900f2610476d2c9f",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "doff",
        "payloadType": "str",
        "x": 50,
        "y": 600,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "14bcdacdbafa055c",
        "type": "inject",
        "z": "900f2610476d2c9f",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "don",
        "payloadType": "str",
        "x": 70,
        "y": 560,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "8a3720bdd399a9c6",
        "type": "inject",
        "z": "900f2610476d2c9f",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "30 16 * * *",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "con",
        "payloadType": "str",
        "x": 50,
        "y": 780,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "4c6359f5e1bf8a59",
        "type": "inject",
        "z": "900f2610476d2c9f",
        "name": "",
        "props": [
            {
                "p": "payload"
            },
            {
                "p": "topic",
                "vt": "str"
            }
        ],
        "repeat": "",
        "crontab": "00 23 * * *",
        "once": false,
        "onceDelay": 0.1,
        "topic": "",
        "payload": "coff",
        "payloadType": "str",
        "x": 50,
        "y": 820,
        "wires": [
            [
                "2c8dde1dc9576b81"
            ]
        ]
    },
    {
        "id": "a126c8f2b8395e24",
        "type": "debug",
        "z": "900f2610476d2c9f",
        "name": "",
        "active": true,
        "tosidebar": true,
        "console": false,
        "tostatus": false,
        "complete": "false",
        "statusVal": "",
        "statusType": "auto",
        "x": 630,
        "y": 340,
        "wires": []
    },
    {
        "id": "ee1dcffd99721703",
        "type": "debug",
        "z": "900f2610476d2c9f",
        "name": "",
        "active": true,
        "tosidebar": true,
        "console": false,
        "tostatus": false,
        "complete": "true",
        "targetType": "full",
        "statusVal": "",
        "statusType": "auto",
        "x": 840,
        "y": 280,
        "wires": []
    },
    {
        "id": "36b93868f3cfa2da",
        "type": "hue-bridge",
        "name": "HueMagic Bridge",
        "bridge": "192.168.1.23",
        "key": "IwnC3vTIm1b5YA9x1YgVMeFJ9MqvWBHPlJHFejnf",
        "disableupdates": false
    }
]
```

**Comment by UweNB** [comment]
Hoffe, es hilft: 

bei A:
10.1.2022, 19:53:08node: 32d6c7a0f2a8665a
PT2262 : msg.payload : string[3]
"aon"

10.1.2022, 19:53:08node: 37c1d192b62ce375
PT2262 : msg : Object
object
status: object
rssi: 6
topic: "PT2262"
payload: true
_msgid: "4b87ab016d58d525"
channel: "a"

bei A off

10.1.2022, 19:54:00node: 32d6c7a0f2a8665a
PT2262 : msg.payload : string[4]
"aoff"

10.1.2022, 19:54:00node: 37c1d192b62ce375
PT2262 : msg : Object
object
status: object
rssi: 6
topic: "PT2262"
payload: false
_msgid: "b6523f13b4d28dcb"
channel: "a"

**Reply by Foddy** [reply]
Okay, möglicherweise lag es an dem *msg.topic*. Kannst du es bitte mit dem neuesten Fix (v4.0.3) nochmal versuchen? Vermutlich ist es damit gelöst.

**Reply by UweNB** [reply]
Klaro! Muss aber erstmal die Enkel in den Schlaf kriegen :) Melde mich dann …

**Reply by Foddy** [reply]
Klar, kein Stress :)

**Reply by UweNB** [reply]
So Foddy, 4.0.3 tut es wieder perfekt! Hatte mittelschwere Panikattacken, dass mein amateur-Flow es nicht tut :) Danke fürs fixen!

**Reply by Foddy** [reply]
Haha, super, freut mich!

**Reply by UweNB** [reply]
Kannst Du kurz populärwissentschaftlich sagen, was da war mit msg.object für mein Verständnis? Danke!

**Reply by Foddy** [reply]
Klar, eigentlich ganz einfaches Thema. Manche externen Nodes schicken an die HueMagic-Nodes einen zusätzlichen Befehl bzw. Parameter, den `msg.topic`-Parameter.

Manchmal ist der leer, manchmal beinhaltet er einen Node-spezifischen Wert. Die HueMagic-Nodes wiederum haben versucht diesen Wert als Ressourcen-ID zu verwenden – also dachten es sein eine alternative Konfiguration des Lichts z. B. – was natürlich zu einem Fehler geführt hat, weil die Ressource auf der Bridge nicht gefunden wurde.

Ich habe jetzt also ein paar zusätzliche Überprüfungen programmiert, damit das hoffentlich nicht so bald wieder passiert :P

**Reply by UweNB** [reply]
Danke für die Erklärung! Bisher getestet mit 4.0.3: 4x Stedo (on/off) 3x Leuchte  (on/off/brighness) und Rückmeldung von den BWMs (Motion und Helligkeit) - hat alles Bestens funktioniert! Danke nochmal ...
